_moira.pyx

```diff
@@ -28,14 +28,18 @@ cdef extern from "stdlib.h":
     void free(void * ptr)
 
 class MoiraException(Exception):
+    @property
     def code(self):
-        return self.args[0]
-    code = property(code)
+        return self.args[0][0]
```

Comment:
This doesn't look right to me, why is there the extra array index?

Reply:
After fixing the `raise MoiraException` statement, the extra index is not necessary anymore